crt_portal/cts_forms/templates/forms/snippets/sortable_table_heading.html

```diff
@@ -0,0 +1,18 @@
+{% load static %}
+
+
+<th scope="col" class="{% if sort_url %}sort-cell{% endif %}">
+  {% if sort_url %}
+    <a
+      href="{{sort_url}}"
+      class="sort-link display-block {% if is_descending %}sort-up{% else %}sort-down{% endif %}"
```

Comment:
This is where I'd add the `white-space: nowrap` rule, if you wanted to try it.

Reply:
this worked great! thanks 😄